sport/app/football/views/wallchart/wallchart.scala.html

```diff
@@ -1,72 +1,125 @@
-    case knockoutStage: _root_.football.model.KnockoutSpider => {
-        <div class="facia-container facia-container--layout-content">
-            <div class="container">
-                <div class="facia-container__inner">
-                    <div class="container__border hide-on-mobile"></div>
-                    <h3 class="container__title">
-                        <span class="container__title__label u-text-hyphenate">knockout stage</span>
-                    </h3>
-                    <div class="container__body">
-                        <div data-link-name="@competition.fullName knockout chart">
-                            @football.views.html.wallchart.knockoutSpider(competition, knockoutStage)
-                        </div>
+@(competition: Competition, competitionStages: List[CompetitionStageLike], next: Option[pa.FootballMatch])(implicit request: RequestHeader)
+<div class="l-side-margins wc-overview">
+
+    @* Remove once switch expires (temporary addition for world cup 2018 - competition '700'). *@
+    @if(Switches.WorldCupNextMatchSwitch.isSwitchedOn && competition.id == "700") {
+        <div class="facia-container facia-container--layout-content facia-header">
+            <div class="facia-header__inner gs-container">
+                <div class="facia-header__column">
+                    @fragments.inlineSvg("world_cup_badge", "badges", List("world__cup-svg", "world__cup-badge"))
+                    <h1 class="container__title"><span class="container__title__label">World cup 2018</span></h1>
+                </div>
+
+                @next.map { nextMatch =>
+                <div class="facia-header__column">
+                    <div class="wc-match__date">
+                        <span class="wc-label wc-label__title">Next match</span>
+                        <span class="wc-label">@nextMatch.date.toString("HH:ss EE d MMM z")</span>
+                    </div>
+                </div>
+                <div class="facia-header__column">
+                    <div class="wc-match__badges">
+                        <span class="wc-label wc-badge__flag"></span>
+                        <span class="wc-label wc-badge__flag"></span>
                     </div>
+                    <div class="wc-match__teams">
+                        <span class="wc-label wc-label__title">@nextMatch.round.name.getOrElse(nextMatch.round.roundNumber)</span>
+                        <span class="wc-label">@pa.TeamCodes.codeFor(nextMatch.homeTeam) vs @pa.TeamCodes.codeFor(nextMatch.awayTeam)</span>
+                    </div>
+                </div>
+                }.getOrElse{
+                <div class="facia-header__column"></div>
+                <div class="facia-header__column"></div>
+                }
+
+                <div class="facia-header__column">
+                    <span class="wc-player-image wc-player-image__offset"><img src="https://uploads.guim.co.uk/2018/05/24/world_cup_cut_out_1.png"></span>
```

Comment:
Is it worth adding alt tags to thee images? If they're just decorative, consider adding `role="presentation"` to hide them from screen readers